File: sg02/Assets/Scripts/UI/StartMenu/StartMenuView.cs
using UnityEngine;
using System.Collections;

public class StartMenuView : MonoBehaviour 
{

    public GameObject m_btnStartGame;
    public GameObject m_btnLoadGame;
    public GameObject m_btnSetting;
    public GameObject m_btnQuit;

	// Use this for initialization
	void Start () 
    {
        StartMenuControl.Instance.Initialize(this);
	}

    void OnDestroy()
    {
        StartMenuControl.Instance.UnInitialize();
    }
}

File: sg02/Assets/Resources/Lua/UI/StartMenu/StartMenuView.lua
module(..., package.seeall);

m_btnStartGame = nil
m_btnLoadGame = nil
m_btnSetting = nil
m_btnQuit = nil


function Initialize(viewPanel)

    m_btnStartGame = viewPanel.transform:FindChild("Buttons/Start_game").gameObject
    m_btnLoadGame = viewPanel.transform:FindChild("Buttons/Load_game").gameObject
    m_btnSetting = viewPanel.transform:FindChild("Buttons/Setting_game").gameObject
    m_btnQuit = viewPanel.transform:FindChild("Buttons/Quit_game").gameObject

end